### PR TITLE
DOC Use 'Axes' in _axes.py docstrings

### PR DESCRIPTION
**Pull request by lucyleeow**
## PR Summary
Towards #18726
Use Axes instead of axes in `_axes.py`.
## PR Checklist

- [N/A] Has pytest style unit tests (and `pytest` passes).
- [N/A] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [N/A] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [N/A] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [N/A] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [N/A] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -5345,10 +5345,10 @@ def imshow(self, X, cmap=None, norm=None, aspect=None,
 
         origin : {'upper', 'lower'}, default: :rc:`image.origin`
             Place the [0, 0] index of the array in the upper left or lower
-            left corner of the axes. The convention (the default) 'upper' is
+            left corner of the Axes. The convention (the default) 'upper' is
             typically used for matrices and images.
 
-            Note that the vertical axes points upward for 'lower'
+            Note that the vertical Axes points upward for 'lower'
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py:5351]: Actually this should have been a singular:
```suggestion
            Note that the vertical axis points upward for 'lower'
```

**Review by timhoffm — COMMENTED** [review]
Thanks. Looks good, with one exception that's already been wrong on our side.

File: lib/matplotlib/axes/_axes.py (hunk 1 above)
Comment (lucyleeow) [reply lib/matplotlib/axes/_axes.py:5351]: Thank you, I should have caught that!

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Thanks @lucyleeow! Congratulations on your first PR to Matplotlib :tada: We hope to hear from you again.

**Pull request merged**